internal/cli/atlas/quickstart/description.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package quickstart
+
+const (
+	QuickStart     = "Create and access an Atlas Cluster."
+	LongQuickStart = "This command creates a cluster, adds your public IP to the atlas access list and creates a db user to access your MongoDB instance. "
+)
```

Comment:
I'm just remembering we don't do description files anymore so let's inline these